index.html

```diff
@@ -0,0 +1,144 @@
+           is a five-city event starting in Kigali on February 24, 2022. Accra is the
+           next stop on March 25, followed by Lusaka (May), Government complex (June), and 
+           Ndola (September). The 2021 conference was on November 8-13.
+        </p>
+       <div class="heads"><span id="next-class"
+        >
+        <span id="start-date">13.04.2022.(THU)~ 30.09.2022.(Fri)</span></span>
+      <span id="location">afrotech global Summit, Virtual</span>
+    </div> 
+      </div>
+    </header>
+    <main>
+      <section id="main-curriculum">
+        <h2>Main Programme</h2>
+        <div class="red-line"></div>
+        <div id="languages-container">
+          <div class="languages">
+            <span>
+              <i class="fa-solid fa-head-side-headphones"></i>
+            </span>
+            <h3>AUDIBLE</h3>
+            <p>Create audible devices.</p>
+          </div>
+          <div class="languages">
+            <span>
+              <i class="fa-solid fa-microphone"></i>
+            </span>
+            <h3>SPEECH</h3>
+            <p>future of text to speech</p>
+          </div>
+          <div class="languages">
+            <span>
+              <i class="fa-solid fa-microchip-ai"></i>
+            </span>
+            <h3>AI</h3>
+            <p>future of robotics</p>
+          </div>
+          <div class="languages">
+            <span>
+              <i class="fa-solid fa-microchip"></i>
+            </span>
+            <h3>CPU</h3>
+            <p>Develop powerful processing units</p>
+          </div>
+          <div class="languages">
+            <span>
+              <i class="fa-solid fa-car-mirrors"></i>
+            </span>
+            <h3>REMOTE</h3>
+            <p>remote control of delivery devices</p>
+          </div>
+        </div>
+        <button type="button">Join our course 2022</button>
+        <span id="see-program">SEE THE WHOLE PROGRAM</span>
+      </section>
+      <section id="speakers">
+        <h2>Featured speakers</h2>
+        <div class="red-line"></div>
+      </section>
+      <section id="partners">
+        <h2>Partners</h2>
+        <div class="red-line"></div>
+        <div id="partners-container">
+          <span><i class="fa-brands fa-youtube"></i>youtube</span>
+          <span><i class="fa-brands fa-facebook-messenger"></i>Messenger</span>
+          <span><i class="fa-solid fa-spider"></i>Spider</span>
+          <span><i class="fa-solid fa-earth-africa"></i>AU</span>
+          <span><i class="fa-solid fa-microchip"></i>Timer</span>
+        </div>
+      </section>
+    </main>
+    <footer>
+      <div class="logo">
+        <span id="footer-logo">
+          <i class="fa-brands fa-artstation fa-2x"></i>
+        </span>
+        <div id="logo-name">
+          <span>Afrotech</span>
+        </div>
+      </div>
+      <p id="footer-p">
+       Afrotech Summit 2021. All &copy; Rights Reserved
+      </p>
+    </footer>
```

Comment:
- [ ] [OPTIONAL] Kindly ensure to reduce the space in between the logo and the text on the footer of the mobile for both the `Homepage` and the `About page`.

![Screenshot from 2022-03-11 21-07-02](https://user-images.githubusercontent.com/58692590/157954128-e88e077d-8586-42c1-945c-f9c88ed869ef.png)